docs/old_norse.rst

```diff
@@ -315,3 +314,38 @@ Old Norse, like other ancient Germanic languages, is highly inflected. With the
 
     Out[6]: 'þessa'
 
+Old Norse noun declension
+=========================
+
+Old Norse nouns vary according to case (nominative, accusative, dative, genitive), gender (masculine, feminine, neuter) and number (singular, plural). Nouns are considered either weak or strong.
+Weak nouns have a simpler declension than strong ones.
+
+If you want a simple way to define the inflection of an Old Norse noun, you can do as follows:
+
+.. code-block:: python
+
+    In[1]: from cltk.inflection.utils import Noun, Gender
+
+    In[2]: sumar = [["sumar", "sumar", "sumri", "sumars"], ["sumur", "sumur", "sumrum", "sumra"]]
+
+    In[3]: noun_sumar = Noun("sumar", Gender.neuter)
+
+    In[4]: noun_sumar.set_declension(sumar)
+
+
+To decline a noun and if you know its nominative singular, genitive singular and nominative plural forms, you can use the following functions.
+
++--------+-------------------------------+------------------------------+----------------------------+
```

Comment:
Being narrowed down to just nouns and just for ON, I can better imagine the use for this -- essentially it's an ON noun type.

l will accept this PR, however I would like to see some work which uses these. For example, what kind of data set can you use -- a lexicon? And do you expect this to help in parsing texts? or for other reasons, too?

Reply:
Hey @kylepjohnson, [this document](https://github.com/cltk/old-norse-lemmatizer/blob/master/README.md) may summarize what I expect to do with such code.